tests/unit/replicationconfig.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage,
```

Comment:
Can we fix the license headers in few files?

Reply:
Fixed